app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,52 @@
+class BlogsController < ApplicationController
+  before_action  :set_blog, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @blogs = Blog.all
+  end
+
+  def show
+  end
+
+  def new
+    @blog = Blog.new
+  end
+
+  def edit
+  end
+
+  def create
+    @blog = Blog.new(blog_params)
+    @blog.author = current_user
+    if @blog.save
+      redirect_to @blog, notice: 'Blog was successfully created.' 
+    else
+      render :new 
+    end
+  end
+
+  def update
+    if @blog.update(blog_params)
+      redirect_to @blog, notice: 'Blog was successfully updated.' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.